browser-test/testData.json

```diff
@@ -0,0 +1,7 @@
+{
+  "testuser": {
+    "email": "testuser@openwisp.org",
+    "password": "testuser",
+    "organization": "default"
```

Comment:
this assumes there's an org named "default" which may not be true, so the script can fail because of this, which is ok if we get a good error message, but no error message is being given.

If you try to delete the default organization you should then see:

```
TimeoutError: Waiting element to become stale
Wait timed out after 5084ms
```

I lowered the timeout to 5 secs, I guess it could be lower as well.

Please make sure that if this case happens a proper error message is printed out, also double check this timeout thing which seems too big, 20/30 seconds is definitely annoying to work with, I put it to 5 secs but I am wondering why do we need such a long timeout in the first place? Ensure it's as low as possible, tests should fail fast if there's an issue like that. The faster they fail the better.